src/app/content/components/BookBanner.tsx

```diff
@@ -32,6 +32,7 @@ const gradients: {[key in BookWithOSWebData['theme']]: string} = {
   'gray': '#97999b',
   'green': '#9cd14a',
   'light-blue': '#1EE1F0',
+  'midnight': '#003e52',
```

Comment:
this looks like the same color as the theme, do we not want this one to have a gradient?

Reply:
I didn't notice those were the light-ends of the gradients. We don't seem to have a midnight gradient set up in osweb (it's using the blue gradient for midnight-theme books), but #005a72 looks nice to me:
![image](https://github.com/user-attachments/assets/9aa36c64-2e40-459d-9e3f-0deff6b8b9b2)